beta/examples/tensorflow/segmentation/train.py

```diff
@@ -45,7 +44,8 @@ def get_argument_parser():
                                         save_checkpoint_freq=False,
                                         export_args=False,
                                         print_freq=False,
-                                        dataset_type=False)
+                                        dataset_type=False,
+                                        cpu_only=False)
```

Comment:
Do you figure out the root of the training issue on CPU?

Reply:
I tried. The problem sometimes is not reproduced. I could try to fix it again.